.github/CODEOWNERS

```
@@ -185,6 +185,7 @@ tests/foreman/ui/test_registration.py @SatelliteQE/team-rocket
 tests/foreman/ui/test_settings.py @SatelliteQE/team-rocket
 tests/foreman/ui/test_smartclassparameter.py @SatelliteQE/team-rocket
 tests/foreman/ui/test_subnet.py @SatelliteQE/team-rocket
+tests/new_upgrades/test_discovery.py @SatellitQE/team-rocket
```

```suggestion
tests/new_upgrades/test_discovery.py @SatelliteQE/team-rocket
```
A typo needs to be fixed.